kotlinx-coroutines-core/jsWasm/src/flow/internal/SafeCollector.kt

```diff
@@ -29,4 +29,4 @@ internal actual class SafeCollector<T> actual constructor(
 
     public actual fun releaseIntercepted() {
     }
-}
+}
```

Comment:
This is a hint that we use newlines at the end of each file :) 

https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

Reply:
Fixed